src/rgw/rgw_sal_daos.cc

```diff
@@ -407,6 +409,11 @@ int DaosBucket::open(const DoutPrefixProvider* dpp) {
 
   ret = dfs_mount(store->poh, coh, O_RDWR, &dfs);
   ldpp_dout(dpp, 20) << "DEBUG: dfs_mount ret=" << ret << dendl;
+#else
+  // How can i get the pool name?
+  const auto& daos_pool = g_conf().get_val<std::string>("daos_pool");
```

Comment:
Maybe we can store the pool name in a variable in the store, then we do not need `store->poh`

Reply:
@tperkins-seagate Could you address this comment?